tests/onnx/quantization/test_min_max.py

```diff
@@ -1,222 +0,0 @@
-# Copyright (c) 2023 Intel Corporation
```

Comment:
Why it was completely deleted?

Reply:
This is intended to test `get_reduction_shape()` for ONNX. In this PR all backends switch to common `get_reduction_axes()`, which is tested in a common test.